api/BaseAudioContext.json

```diff
@@ -417,10 +473,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": "53"
+              "version_added": "52"
```

Comment:
This change is not accidental, it's from mdn-bcd-collector results. It matches https://developer.mozilla.org/en-US/docs/Web/API/ConstantSourceNode#browser_compatibility.